docs/user.md

```diff
@@ -612,6 +612,8 @@ gallery:
     - http://i.imgur.com/o9r19kD.jpg "Dubai"
     - https://example.com/orignal.jpg https://example.com/thumbnail.jpg "Sidney"
 comments: false
+hideMeta: true
```

Comment:
use `meta` and `actions` variable names to have the same code style as `comments`. so update your if condition with `(post.actions === undefined || post.actions)` to still show the meta and actions if variables are not declared in the front-matter.